package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-node-azure-pack",
-    "version": "1.5.0",
+    "version": "1.6.0",
```

Comment:
The version also appears twice in package-lock.json. Can you bump that by running `npm i` in your workspace?

Reply:
Booooo I usually `npm version minor` which handles this, but I was lazy and it bit me